OneCommunityGlobal/HighestGoodNetworkApp
# fix the bluesquare cannot show bug

# Description
I cannot access some of the userprofile pages, example, "Yiyun Tan" on dev and hit below error.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/5071040/4f49c99b-4227-4525-9cdd-a1c505cf10e1)

## Related PRS (if any):
no

## Main changes explained:
Others are files type changes are automatically done by pulling latest development branch. Which is changing from CRLF to LF, see more discussion on slack coding-problem channel.
I just add an conditional for render `div`

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as any user
4. verify "Yiyun Tan"'s profile can be accessed on local but cannot on dev.
